game/scripts/npc/heroes/io.txt

```diff
@@ -5,15 +5,18 @@
   //=================================================================================================================
   "npc_dota_hero_wisp"
   {
-    "Ability16"                                           "special_bonus_hp_900"
-    "Ability17"                                           "special_bonus_unique_wisp_7" // replaces special_bonus_unique_wisp_4
+    "Ability16"                                           "special_bonus_unique_wisp_9"
+    "Ability17"                                           "special_bonus_unique_wisp_2" // replaces special_bonus_unique_wisp_4
```

Comment:
Remind me to test if this works. Otherwise we need to find a new replacement for Attack tether allies, Scepter now gives Spirits Slow

Reply:
Also it could be a good idea to // things you're using for context so we don't get a million errors every time valve changes a talent.